wgpu-core/src/instance.rs

```diff
@@ -552,6 +555,7 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
 
         wgt::Limits {
             max_bind_groups: (limits.max_bound_descriptor_sets as u32).min(MAX_BIND_GROUPS as u32),
+            _non_exhaustive: wgt::NonExhaustive,
```

Comment:
Let's be consistent on whether we include stuff from `wgt` or not. My preference is to just use `wgt::` everywhere.